backend/python/app/resources/user_dto.py

```diff
@@ -6,9 +6,9 @@ def __init__(
         last_name,
         email,
         role,
-        resume,
-        profile_pic,
-        approved_languages,
+        resume=None,
```

Comment:
thanks for updating this - these DTO files are for the REST endpoint. They'll eventually be nuked with REST stuff once everything is wrapped up